src/Network/GRPC/Server/Settings.hs

```diff
@@ -0,0 +1,39 @@
+module Network.GRPC.Server.Settings
+  ( ServerSettings(..)
+  , defaultServerSettings
+  ) where
+
+import Data.Default
+
+import Network.GRPC.Common.HTTP2Settings
+
+-- | Settings affecting the server's concurrency and HTTP\/2 parameters
+data ServerSettings = ServerSettings {
+      -- | Number of threads that will be spawned to process incoming frames
+      -- on the currently active HTTP\/2 streams
+      --
+      -- This setting is specific to the
+      -- [http2](https://hackage.haskell.org/package/http2) package's
+      -- implementation of the HTTP\/2 specification.
+      --
+      -- __Note__: If a lower 'serverInitialConnectionWindowSize' is desired, the
+      -- number of workers should be increased to avoid a potential HTTP\/2
+      -- control flow deadlock.
+      serverNumberOfWorkers :: Word
```

Comment:
Is this value only relevant to the server? The client runs the same frame receiver loop right..? Do the incoming frames get handled differently?

Reply:
The concept of worker threads is specific to servers in `http2`, so this makes sense here I believe